### ISSUE TITLE
Some tests regarding format-value fail with German language settings

### ISSUE THREAD

**Issue by rstropek** [aspnet/Blazor#461]
`Render_BuiltIn_BindToInputText_WithFormatFromProperty_WritesAttributes` fails when building Blazor on a Windows machine with German language settings:

```
Inner exception: Assert.Equal() Failure
  Expected: 01/01/2018
  Actual:   01.01.2018
```

The *Actual* value reflects the date settings on the machine I am building Blazor on.

**Comment by rstropek** [comment]
Just saw that the same applies to `Render_BindToElementFallback_WithFormat_WritesAttributes`

**Comment by rynowak** [comment]
Ooh thanks, this is a good one. 

Looks like we forgot to pass in a culture.

**Comment by SteveSandersonMS** [comment]
Fixed in #470
